### PR TITLE
[MOSIP-29167]added reusable workflow

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

**Review by syedsalman3753 — APPROVED** [review]

**Comment by ckm007** [comment]
Can we get this approved once by @sowmya695  as there are pom changes in the same?

**Review by sowmya695 — APPROVED** [review]
Approved,Please test in dev1 env once and then i will approve the release changes

File: .github/workflows/release-changes.yml
Comment (ckm007) [inline .github/workflows/release-changes.yml]: add webhook_url and action_pat as secrets.

**Pull request merged**
